integrationtests/elasticsearch-indexmanager-it/src/test/java/org/infinispan/query/it/ClusteredCacheWithElasticsearchIndexManagerIT.java

```diff
@@ -17,6 +18,10 @@
  */
 @Test(groups = "functional", testName = "query.blackbox.ClusteredCacheWithElasticsearchIndexManagerIT")
 public class ClusteredCacheWithElasticsearchIndexManagerIT extends ClusteredCacheTest {
+    @Override
+    public Object[] factory() {
```

Comment:
The fix makes me sad. But such is life after you've adopted TestNG.

Reply:
We make the decision whether to run the factory method or not in `MultipleCacheManagersTest`. I could have changed it to never throw an exception and just ignore the `factory()` method from super-classes, if that makes you feel better @anistor